pallets/emergency-shutdown/src/lib.rs

```diff
@@ -61,10 +61,14 @@ pub mod pallet {
     pub enum Event<T: Config> {
         /// Toggled Pallet
         /// [flag]
-        ToggledPallet(bool),
+        ToggledPallet { pallet_idx: u8, updated_flag: bool },
         /// Toggled Call
         /// [flag]
-        ToggledCall(bool),
+        ToggledCall {
+            pallet_idx: u8,
+            call_idx: u8,
+            updated_flag: bool,
+        },
```

Comment:
@cr4pt0 Thanks for your pr, currently our events are tuples, can you change it and add detailed comments, to keep consistent with others